### PR TITLE
Fix CV tab spacing

### PR DESCRIPTION
**Pull request by zephraph**
Okay, so this is largely done. 

Big issue is that the md breakpoint is behaving very oddly. @mzikherman could you help me look into this?

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit f0e037df02b1a98346f703a13b22f11c8e814ea7

https://deploy-preview-999--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
@zephraph sure what's up! I couldn't tell immediately if anything looked odd from the preview link. Happy to pair or do some sleuthing on my own if you want to point me to something thats not working/looks off?

File: src/Styleguide/Elements/Join.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React, { SFC } from "react"
```
Comment (mzikherman) [inline src/Styleguide/Elements/Join.tsx]: aw yea 😎

File: src/Styleguide/Utils/BreakpointVisualizer.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      this.setState({
+        changed: true,
+      })
+    }
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        {!navigator.userAgent.match(/Chromatic/) && (
```
Comment (mzikherman) [inline src/Styleguide/Utils/BreakpointVisualizer.tsx]: what is `navigator` here? Is that some global or something?

Comment (zephraph) [reply src/Styleguide/Utils/BreakpointVisualizer.tsx]: Yep. It's generally on the window... might need to check for its existence I guess.

**Comment by zephraph** [comment]
Actually, the grid issue doesn't seem to be happening anymore.... Just one minor spacing thing that I'm about to fix and I guess it'll be good?

**Comment by mzikherman** [comment]
LGTM!

**Comment by zephraph** [comment]
Okay, spacing is finalized, this is good to go. 

@mzikherman I'd note that the breakpoint visualizer is a debug component. When you build the app in dev mode it'll show you what breakpoint you're on. It disappears after a few seconds. It doesn't show up in prod mode (as such, it won't show in the story previews).

**Pull request merged**